tests/IntegrationTests/WorkflowExecutor.IntegrationTests/Features/PayloadApi.feature

```diff
@@ -42,6 +42,24 @@ Scenario: Get all payloads from API - no payloads
     Then I will get a 200 response
     And I can see no Payloads are returned
 
+@PayloadSearch
+Scenario Outline: Get all payloads from API - Test search query parameters
+    Given I have an endpoint /payload/<search_query>
+    And I have 10 Payloads
+    When I send a GET request
+    Then I will get a 200 response
+    And Search is working correctly for the <search_count> payloads
+    Examples:
+    | search_query                                                           | search_count |
```

Comment:
Should the examples table not be aligned 
e.g 
| search_query | search_count|

Reply:
Yeah they are aligned, just Github being weird